src/Compilers/CSharp/Portable/Symbols/Tuples/TupleErrorFieldSymbol.cs

```diff
@@ -77,15 +97,17 @@ public override ImmutableArray<SyntaxReference> DeclaringSyntaxReferences
         {
             get
             {
-                return GetDeclaringSyntaxReferenceHelper<CSharpSyntaxNode>(_locations);
+                return _isImplicitlyDeclared ?
+                    ImmutableArray<SyntaxReference>.Empty :
+                    GetDeclaringSyntaxReferenceHelper<CSharpSyntaxNode>(_locations);
```

Comment:
It seems like `_locations` should be empty if `_isImplicitlyDeclared`. If so, call `GetDeclaringSyntaxReferenceHelper<>()` always.